src/mip/mip_logging.c

```diff
@@ -0,0 +1,79 @@
+
+#include "mip_logging.h"
+
+#include <stddef.h>
+
+////////////////////////////////////////////////////////////////////////////////
+///@brief Global logging callback. Do not access directly
+mip_log_callback _mip_log_callback = NULL;
```

Comment:
Names beginning with underscore are reserved in the global scope. I suggest moving it to the end instead.